Marlin/src/HAL/shared/eeprom_i2c.cpp

```diff
@@ -50,29 +58,31 @@ static void eeprom_init() {
 }
 
 void eeprom_write_byte(uint8_t *pos, unsigned char value) {
-  unsigned eeprom_address = (unsigned) pos;
+  unsigned eeprom_address = (unsigned)pos;
 
-  eeprom_init();
+  TERN(TARGET_LPC1768,,eeprom_init());
```

Comment:
Your LPC version skipped this init. Is that due to the way its EEPROM works?

Reply:
`eeprom_init()` calls `Wire.begin()` which initialises the hardware peripheral, doing so on every byte is incredibly slow (even triggers the watchdog on large block writes), it was moved to `PersistentStore.access_start()` so it was only called once per block write.

Reply:
Yeah, it seemed weird to find it in those places, but I wanted to be sure it wasn't an intentional hack.

Reply:
Aaaand… I also wanted to make sure the ones included in 4b16fa3272032a66200515750608f35dffbf1c09 originally were not intentional.

With those lines now removed, this patch is a major fix for I2C EEPROM performance across all the boards that use them!